compute_tools/src/spec.rs

```diff
@@ -697,3 +697,72 @@ pub fn handle_extension_neon(client: &mut Client) -> Result<()> {
 
     Ok(())
 }
+
+#[instrument(skip_all)]
+pub fn handle_migrations(client: &mut Client) -> Result<()> {
+    info!("handle migrations");
+    let migrations = [
+        "ALTER ROLE neon_superuser BYPASSRLS",
+        r#"
+DO $$
+DECLARE
+    role_name text;
+BEGIN
+    FOR role_name IN SELECT rolname FROM pg_roles WHERE pg_has_role(rolname, 'neon_superuser', 'member')
```

Comment:
I've tried running this manually on staging via `cloud_admin` and it immediately errors out with:

```
ERROR:  can't ALTER neon_superuser
CONTEXT:  SQL statement "ALTER ROLE neon_superuser INHERIT"
PL/pgSQL function inline_code_block line 7 at EXECUTE
```

Reply:
Up: I see now corresponding change with `if (RoleIsNeonSuperuser(role_name) && !superuser())` to allow it.